lib/parser/epoch.ex

```diff
@@ -7,7 +7,7 @@ defmodule DateTimeParser.Epoch do
   def parse(%{"seconds" => raw_seconds, "subseconds" => raw_subseconds}) do
```

Comment:
Since these are moved under the `parser` directory, does it make sense to add the `Parser` namespace to the module name?

Reply:
thought about it but skipped. I should probably namespace it.